schema/vulntology-json-schema-1.0-draft.json

```
@@ -42,22 +42,26 @@
                 }
             ],
             "properties": {
-                "hasEnumeration": {"$ref": "#/definitions/ProductEnumeration"},
+                "hasEnumeration": {
+                    "type": "array",
+                    "$ref": "#/definitions/ProductEnumeration"
```

If the goal is to have items of the `ProductEnumeration` type, then the schema should be:
```suggestion
                    "type": "array",
                    "items": {
                        "$ref": "#/definitions/ProductEnumeration"
                    }
```